tests/integration/test_orders_api.py

```diff
@@ -536,6 +537,171 @@ async def test_download_asset_md(tmpdir, session):
     assert Path(filename).name == 'metadata.json'
 
 
+@respx.mock
+@pytest.mark.asyncio
+async def test_checksum_success(tmpdir, order_description, oid, session):
+    # Mock an HTTP response for download
+    order_description['state'] = 'success'
+    dl_url1 = TEST_DOWNLOAD_URL + '/asset1'
+    dl_url2 = TEST_DOWNLOAD_URL + '/asset2'
+    dl_url3 = TEST_DOWNLOAD_URL + '/manifest'
+    order_description['_links']['results'] = [{
+        'location': dl_url1
+    }, {
+        'location': dl_url2
+    }, {
+        'location': dl_url3
+    }]
+    get_url = f'{TEST_ORDERS_URL}/{oid}'
+    get_order_response = httpx.Response(HTTPStatus.OK, json=order_description)
+    respx.get(get_url).return_value = get_order_response
+
+    # The first asset in the order:
+    # A 1-byte binary file containing the value "1".
+    asset1_content = b"1"
+    asset1_response = httpx.Response(HTTPStatus.OK,
+                                     content=asset1_content,
+                                     headers={
+                                         'Content-Type':
+                                         'image/tiff',
+                                         'Content-Disposition':
+                                         'attachment; filename="asset1.tif"'
+                                     })
+    respx.get(dl_url1).return_value = asset1_response
+
+    asset2_response = httpx.Response(HTTPStatus.OK,
+                                     json={
```

Comment:
This is just an example, it could be anything. As an example, the first asset used in this test is a tif file (so not even json). This works because the hashkey is calculated based on the contents of the file, regardless of what it it.

Reply:
ok, cool. thanks. would you be willing to change the json to something like `{'foo': 'bar'}` to highlight the fact that the content of the json doesn't matter? That will be a little clearer than having the content be related to checksums.